scripts/lintinstall/index.js

```diff
@@ -0,0 +1,17 @@
+const fs = require('fs');
+const tslintConfig = require(`${__dirname}/../../tslint.json`);
+const tsConfigLint = require(`${__dirname}/../../tsconfig.lint.json`);
```

Comment:
you should probably use path.join(__dirname, '..', '..', 'tslint.json') to get something working in every os